drakrun/drakrun/lib/postprocessing/__init__.py

```diff
@@ -44,6 +45,15 @@ class PostprocessPlugin(NamedTuple):
         requires=["procmon.log"],
         generates=["process_tree.json"],
     ),
+    PostprocessPlugin(
+        function=build_report,
+        requires=[
+            "metadata.json",
+            "process_tree,json",
+            "apimon.log",
+            "ttps.json",
```

Comment:
Are these all necessary in all cases? For example, I think the code can work without `ttps.json`, it will just miss some information. So there are actually no requirements (metadata.json is implicit and always available).